### PR TITLE
Bump imports to 3.6

### PR DESCRIPTION
**Pull request by Thaodan**

### CODE REVIEW

File: src/imports/qml2/plugins.qmltypes [hunk 1]
```diff
@@ -12,7 +12,7 @@ Module {
         prototype: "QObject"
         exports: [
             "Communi/Irc 3.0",
-            "Communi/Irc 3.2",
+            "Communi/Irc 3.6",
```
Comment (jpnurmi) [inline src/imports/qml2/plugins.qmltypes:15]: i wonder why 3.2 turned to 3.6. did you generate the file with qmlplugindump?

Comment (jpnurmi) [reply src/imports/qml2/plugins.qmltypes:15]: Try something like:
```
QML2_IMPORT_PATH=/path/to/libcommuni/qml/ qmlplugindump -nonrelocatable Communi 3.0 > /path/to/plugins.qmltype
```

**Comment by Thaodan** [comment]
Thanks, fixed.

**Review by jpnurmi — APPROVED** [review]
Thanks, LGTM

**Pull request merged**